src/components/Board.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Grid, Paper } from "@mui/material";
+import { formatNumber, LIMIT } from "../util/utility";
+
+const Board: React.FC = () => {
+  const arr: Array<number> = React.useMemo(
+    () => Array.from({ length: LIMIT }, (_) => 0),
+    []
+  );
+
+  return (
+    <Grid
```

Comment:
wrap it in paper

Reply:
should I do like this <Paper><Grid><Grid/><Paper>?